lib/bots/versionbot.ts

```diff
@@ -33,6 +33,7 @@ import { AlertLevel, LogLevel } from '../utils/logger';
 // Exec technically has a binding because of it's Node typings, but promisify doesn't include
 // the optional object (we need for CWD). So we need to special case it.
 const fsReadFile: (filename: string, options?: any) => Promise<Buffer | string> = Promise.promisify(FS.readFile);
+const fsWriteFile: (path: string, contents: string) => Promise<{}> = Promise.promisify(FS.writeFile);
```

Comment:
Is it worth using `mz` for these functions? They're already promisified and types are provided.